ocaml/xapi/message_forwarding.ml

```diff
@@ -3238,6 +3238,34 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       Sm.assert_session_has_internal_sr_access ~__context ~sr:self;
       Local.SR.set_physical_utilisation ~__context ~self ~value
 
+    let assert_all_pbds_unplugged ~__context ~sr =
```

Comment:
I would prefer to keep the assert function definitions themselves in xapi_sr.ml, and just call them from message_forwarding.ml.

Reply:
Or xapi_sr_operations.ml perhaps.

Reply:
I opted for the second option, it looks like the functions in in `xapi_sr.ml` are only accessible if they are exposed through the datamodel api.